BackofficeBundle/Tests/Manager/NodeManagerTest.php

```diff
@@ -80,6 +80,7 @@ public function testDuplicateNode()
         $siteId = 'fakeSiteId';
         $language = 'fakeLanguage';
         $version = 1;
+        $boDirection = 'h';
```

Comment:
Is there an exploitable constant somewher on a ninterface instead of hardly coding this value ?